UI Test Runner/UI Test Runner/TestViews/FormViewTestView.swift

```diff
@@ -83,19 +111,66 @@ struct FormViewTestView: View {
                 }
             }
     }
-}
-
-private extension FormViewTestView {
-    /// A Boolean value indicating whether the form controls should be shown directly in the form's presenting container.
+    
+    /// Test case selection UI.
+    var testCaseSelector: some View {
+        ScrollView {
+            ForEach(cases) { testCase in
+                Button(testCase.id) {
+                    self.testCase = testCase
+                    map = Map(url: .init(string: testCase.url)!)
+                }
+            }
+        }
+    }
+    
+    /// A Boolean value indicating whether the form controls should be shown directly in the form's
+    ///  presenting container.
     var useControlsInForm: Bool {
         verticalSizeClass == .compact ||
         UIDevice.current.userInterfaceIdiom == .mac ||
         UIDevice.current.userInterfaceIdiom == .pad
     }
-}
-
-private extension URL {
-    static var sampleData: Self {
-        .init(string: <#URL#>)!
+    
+    /// Test conditions for a Form View.
+    struct TestCase: Identifiable {
+        /// The name of the test case.
+        let id: String
+        /// The object ID of the feature being tested.
+        let objectID: Int
+        /// The test data location.
+        let url: String
     }
+    
+    /// Makes a URL a for test data.
+    /// - Parameter id: A portal item ID.
+    /// - Returns: A URL for Form View test data with the given portal item ID.
+    func makeURL(for id: String) -> String {
+        String("https://\(String.formViewTestDataDomain!)/home/item.html?id=\(id)")
+    }
+    
+    /// The set of all Form View UI test cases.
+    var cases: [TestCase] {[
+        .init(id: "testCase_1_1", objectID: 1, url: makeURL(for: String.formViewTestDataCase_1_x!)),
```

Comment:
Good suggestion. I changed it so that the URL is just constructed right within the initializer now. 
I think this makes more sense since `makeURL` isn't used anywhere else and it was only 1 line.